code/modules/surgery/organs/augments_arms.dm

```diff
@@ -20,7 +22,16 @@
 
 	update_icon()
 	SetSlotFromZone()
-	items_list = contents.Copy()
+	create_items(normal_items)
+
+/obj/item/organ/cyberimp/arm/proc/create_items(var/list/arg_items, var/emag = 0)
+	for(var/listitem in arg_items)
+		if(ispath(listitem) && !(locate(listitem) in items_list))
+			var/obj/item/I = new listitem(src)
+			I.AddComponent(/datum/component/magnet_drop, src)
+			if(emag)
+				to_chat(usr, "<span class='notice'>You unlock [src]'s integrated [I.name]!</span>")
```

Comment:
`usr` isn't allowed in most cases outside verbs. You'll need to get the user through some other method, probably by just seeing who's wearing the thing.

(The older code is an example of code being untouched for ages)

Reply:
Got it!

Reply:
"owner" seems to be a good way to fix this.